courses/machine_learning/deepdive2/structured/solutions/3_automl_tables_babyweight.ipynb

```diff
@@ -0,0 +1,487 @@
+{
```

Comment:
<p>Punctuation needed after "Great". Comma after "To make these easy".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/677/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Done</p>